source/retryable-reads/retryable-reads.rst

```diff
@@ -310,18 +313,15 @@ The above requirement can be fulfilled in one of two ways:
    there is no guarantee that the lower versioned server can support the
    original command.
 
-3c. If the retry attempt fails
-''''''''''''''''''''''''''''''
+3c. If a retry attempt fails
```

Comment:
Seems like this should say that we should retry "forever", no?  It's implied but not explicit.

Reply:
I think you dealt with this elsewhere, but please confirm in a comment.

Reply:
Yes, as discussed in Slack, I updated the `Deciding to allow retry, encountering the initial retryable error, and selecting a server` (a super-section of `If a retry attempt fails`) to contain information about the exact retry policy. It now explicitly says

> Drivers MUST then retry the operation as many times as necessary until a retry attempt succeeds or returns a non-retryable error.